app/Resources/translations/default/ShopNotificationsError.xlf

```diff
@@ -527,8 +527,8 @@ File: classes/stock/SupplyOrderDetail.php:339</note>
 File: classes/form/CustomerForm.php:178</note>
       </trans-unit>
       <trans-unit id="b99c284c2a24cf024e5811c89706bf79">
-        <source>The email "%mail%" is already used, please choose another one or sign in</source>
-        <target>The email "%mail%" is already used, please choose another one or sign in</target>
+        <source>The email is already used, please choose another one or sign in</source>
```

Comment:
Why removing the mention of the mail?

Reply:
I think it's because of this issue 
![jonathan-22](https://user-images.githubusercontent.com/26626066/47718673-426d9000-dc49-11e8-94b5-e854c02fe06a.gif)

Reply:
![47718673-426d9000-dc49-11e8-94b5-e854c02fe06a](https://user-images.githubusercontent.com/26626066/47779865-71900a00-dcfa-11e8-809e-55364fad4b54.gif)

Because the message box is too small for many email addresses and the email address is obvious, it is written in the field above :) You can see on the gif what I mean.

Reply:
@jolelievre you were 30 sec faster than me 😆